src/compiler/parse/state/tag.ts

```diff
@@ -465,15 +471,10 @@ function read_attribute_value(parser: Parser) {
 		}];
 	}
 
-	const regex = (
-		quote_mark === "'" ? /'/ :
-			quote_mark === '"' ? /"/ :
-				/(\/>|[\s"'=<>`])/
-	);
-
 	let value;
 	try {
-		value = read_sequence(parser, () => !!parser.match_regex(regex));
```

Comment:
just curious, why is this written as a spread, instead of defining another constant list at the top of the file?

Reply:
No special reason to be honest. Do you want med to define everything as a constant?

Reply:
yea that would be better